### PR TITLE
rule `no-patronum-debug`

### LINKED ISSUE #112: Rule: `no-patronum-debug`
I think this rule can help users to remove forgotten `debug` from production code.

Looks like this rule should be auto-fixable.

### PR DESCRIPTION
**Pull request by raidenmiro**
closes #112

- [x] - basic test-cases
- [x] - implementation
- [x] - ts examples
- [x] - docs

### CODE REVIEW

**Review by sergeysova — APPROVED** [review]

**Comment by sergeysova** [comment]
If you gonna to merge, please squash all commits

File: rules/no-patronum-debug/no-patronum-debug.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+              },
+            },
+          ],
+        });
+      },
+    };
+  },
+};
+
+function extractImportFromPatronum({
```
Comment (igorkamyshev) [inline rules/no-patronum-debug/no-patronum-debug.js:61]: Could we use https://github.com/effector/eslint-plugin/blob/master/utils/extract-imported-from.js for this purpose?

**Review by igorkamyshev — COMMENTED** [review]
Thanks for amazing implementation. I love these docs and the number of tests 💙 

I have one minor question, everything else is great.

File: rules/no-patronum-debug/no-patronum-debug.js (hunk 1 above)
Comment (raidenmiro) [reply rules/no-patronum-debug/no-patronum-debug.js:61]: Yes, sure. I'll fix it.

**Review by igorkamyshev — APPROVED** [review]
Great, thanks 💙 

Will be released in an hour ✅

**Pull request merged**